src/index.html

```diff
@@ -1,13 +1,802 @@
+                      class="testimonials__card-quote"
+                    >
+                  </div>
+
+                  <p class="testimonials__card-text">
+                    AIR raises the agency bar to stratospheric heights on both creative output and client service.
+                  </p>
+                </div>
+
+                <div class="testimonials__card-bottom">
+                  <p class="testimonials__card-name">
+                    Michel Grover
+                  </p>
+
+                  <p class="testimonials__card-position">
+                    Hulu
+                  </p>
+                </div>
+              </div>
+              <!-- #endregion Card 3 -->
+            </div>
+            <!-- #endregion Testimonials Cards -->
+          </div>
+        </section>
+        <!-- #endregion Testimonials -->
+
+        <!-- #region Apply -->
+        <section class="apply">
+          <div class="apply__container">
+            <h3 class="apply__title">
+              Vision, Passion, Results
+            </h3>
+
+            <p class="apply__text">
+              We are sure that first-rate job is possible only if all three components are united.
+            </p>
+
+            <a href="#contact" class="apply__button">
+              Apply
+            </a>
+          </div>
+        </section>
+        <!-- #endregion Apply-->
+
+        <!-- #region Contact -->
+        <section class="contact" id="contact">
+          <div class="contact__container">
+            <!-- region Contact Top-->
+            <div class="contact__top">
+              <h3 class="contact__title">
+                Send us a message
+              </h3>
+              <!-- #region Contact Form -->
+              <div class="contact__form-container">
+                <form action="#" class="contact__form">
+                  <label for="email" class="contact__label">
+                    email
+                  </label>
+                  <input
+                    class="contact__input"
+                    type="email"
+                    name="email"
+                    id="email"
+                    placeholder="Your email"
+                    pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$"
+                    required
+                  >
+
+                  <label for="name" class="contact__label">
+                    name
+                  </label>
+                  <input
+                    class="contact__input"
+                    type="name"
+                    name="name"
+                    id="name"
+                    placeholder="Your name"
+                    minlength="2"
+                    maxlength="20"
+                    required
+                  >
+                  <label for="message" class="contact__label">
+                    message
+                  </label>
+                  <textarea
+                    class="contact__input contact__input--textarea"
```

Comment:
It is a better idea to add required for textarea tag as well, because the form shouldn’t submit empty according to the requirements of the task.